cmd/snap/cmd_buy.go

```diff
@@ -109,7 +109,7 @@ Once completed, return here and run 'snap buy %s' again.`), snap.Name, snap.Name
 
 	// TRANSLATORS: %q, %q and %s are the snap name, developer, and price. Please wrap the translation at 80 characters.
 	fmt.Fprintf(Stdout, i18n.G(`Please re-enter your Ubuntu One password to purchase %q from %q
-for %s. Press ctrl-c to cancel.`), snap.Name, snap.Developer, formatPrice(opts.Price, opts.Currency))
+for %s. Press ctrl-c to cancel.`), snap.Name, snap.Publisher.Username, formatPrice(opts.Price, opts.Currency))
```

Comment:
Do we need a check for `snap.Publisher != nil` here?

Reply:
this is a store snap